openreview/venue/recruitment.py

```diff
@@ -87,25 +87,38 @@ def invite_committee(self,
 
         print('sending recruitment invitations')
         for index, email in enumerate(tqdm(invitees, desc='send_invitations')):
+            profile_emails = []
+            profile = None
+            is_profile_id = email.startswith('~')
+            invalid_profile_id = False
+            if is_profile_id:
+                try:
+                    profile = tools.get_profile(self.client, email)
+                except openreview.OpenReviewException as e:
+                    error_string = repr(e)
+                    if 'ValidationError' in error_string:
+                        invalid_profile_id = True
+                    else:
+                        if error_string not in recruitment_status['errors']:
+                            recruitment_status['errors'][error_string] = []
+                        recruitment_status['errors'][error_string].append(email)
+                        continue
+                if not profile:
+                    invalid_profile_id = True
```

Comment:
should we distinguish between invalid_profile_id and not_found_profile_id? I think we could make that difference.